spec/index.html

```diff
@@ -363,6 +363,35 @@ <h3>Triple Terms and Reification</h3>
 
     <p>Note that a <a>triple term</a> may also have another <a>triple term</a> as an <a>object</a>.</p>
   </section>
+  
+  <section id="section-version-announcement">
+    <h3>RDF version announcement</h3>
+	
+    <p>To allow RDF parsers to error or warn about unsupported RDF versions as early as possible,
+		RDF serialization formats SHOULD allow a version to be specified,
```

Comment:
This is an informative section, so we should avoid BCP14 words like SHOULD and MUST, or move them to a normative section.

Reply:
```suggestion
		RDF serialization formats are expected to allow a version to be specified,
```